stacker/providers/aws/default.py

```diff
@@ -414,6 +415,12 @@ def generate_cloudformation_args(stack_name, parameters, tags, template,
     else:
         args["TemplateBody"] = template.body
 
+    if stack_policy:
+        if stack_policy.url:
```

Comment:
Is this for future code, or am I misunderstanding?  I don't see any mention of `stack_policy.url` in this PR.

Reply:
Also, can you throw some a debug statement or two in this block?  That way we can see if someone is using a stack policy with `-v`

Reply:
Yeah, just for the future, in case we ever support uploading to S3 first. I'll add some debug statements to this.

Reply:
Can you throw a comment in there, just so that if someone sees this they aren't confused in the future?